sdk/eventhub/azure-eventhub/azure/eventhub/amqp/_amqp_message.py

```diff
@@ -208,7 +208,7 @@ def properties(self, value: AmqpMessageProperties) -> None:
         self._properties = value
 
     @property
-    def application_properties(self) -> Optional[Dict[Union[str, bytes], Any]]:
```

Comment:
This typing of Dict[Union[str,bytes], Any] makes mypy angry -- solution was to confine it to just str or just bytes